### PR TITLE
fix: dont stringify payload

### LINKED ISSUE #1317: Send the event data directly to and from the service worker without stringifying
### Scope

Improves an existing behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

Send the event data directly to and from the service worker without stringifying

### PR DESCRIPTION
**Pull request by 95th**
- Fixes #1317

### CODE REVIEW

File: src/mockServiceWorker.js [hunk 1]
```diff
@@ -147,20 +147,22 @@ async function handleRequest(event, requestId) {
   if (client && activeClientIds.has(client.id)) {
     ;(async function () {
       const clonedResponse = response.clone()
-      sendToClient(client, {
+      const body =
```
Comment (kettanaito) [inline src/mockServiceWorker.js]: Let's read the request body as text for now. Removing the payload stringification and changing the request body type are two separate things, we can ship them one by one but isolated.

Comment (95th) [reply src/mockServiceWorker.js]: Removed

**Review by kettanaito — APPROVED** [review]
Great! Yet another improvement 🎉 Thank you

**Comment by kettanaito** [comment]
## Released: v0.44.0 🎉

This has been released in v0.44.0!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v0.44.0)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/0.44.0)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.

**Pull request merged**